src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  width: $starwidth;
```

Comment:
You still hardcode width, which means that you should not set width for stars as elements should be able to dictate the width by themself

Reply:
replaced to flrx and column gap, see 'v3'